nominal/cli/config.py

```diff
@@ -0,0 +1,69 @@
+from __future__ import annotations
+
+import dataclasses
+
+import click
+
+from nominal import _config as _deprecated_config
+from nominal import config
+from nominal.cli.util.global_decorators import global_options
+from nominal.cli.util.verify_connection import validate_token_url
+
+
+@click.group(name="config")
+def config_cmd() -> None:
+    pass
+
+
+@config_cmd.group(name="profile")
+def profile_cmd() -> None:
+    pass
+
+
+@profile_cmd.command("add")
+@click.argument("profile")
+@click.option("-u", "--base-url", default="https://api.gov.nominal.io/api", prompt=True)
+@click.option("-t", "--token", required=True, prompt=True, help="bearer token or api key")
+@global_options
+def add_profile(profile: str, base_url: str, token: str) -> None:
+    """Add or update a profile to your Nominal config"""
+    cfg = config.NominalConfig(profiles={}, version=2)
+    try:
+        cfg = config.NominalConfig.from_yaml()
+    except FileNotFoundError:
+        pass
+    validate_token_url(token, base_url)
+    new_cfg = dataclasses.replace(cfg, profiles={**cfg.profiles, profile: config.ConfigProfile(base_url, token)})
+    new_cfg.to_yaml()
+    click.secho(f"Added profile {profile} to {config.DEFAULT_NOMINAL_CONFIG_PATH}", fg="green")
```

Comment:
```suggestion
    click.secho(f"Added profile {profile} to {config.DEFAULT_NOMINAL_CONFIG_PATH}", fg="green")
```

Not a big deal, but the messaging here may be a bit confusing for a user who means to update a profile. We do know whether the profile already exists or not, so could message it accordingly.

Reply:
the suggestion here is empty/no change - what did you mean to suggest instead? makes sense what you're saying

Reply:
Something like `Added profile {profile} to {...}` vs `Updated profile {profile} in {...}`, depending on what has been done. Maybe too nitpicky, and can just always say "Updated profile X in F".

Reply:
I'll say "Wrote profile ..." as it's clear I think in both cases